diff --git a/node_modules/@ap.cx/react-fullpage/dist/index.es.js b/node_modules/@ap.cx/react-fullpage/dist/index.es.js
index 2628663..c19fef9 100644
--- a/node_modules/@ap.cx/react-fullpage/dist/index.es.js
+++ b/node_modules/@ap.cx/react-fullpage/dist/index.es.js
@@ -307,9 +307,10 @@ var Fullpage = /*#__PURE__*/function (_PureComponent) {
       if (!this.ticking) {
         window.requestAnimationFrame(function () {
           // update count
-          _this3.viewportHeight = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
-          _this3.fullPageHeight = _this3.fullpageRef.current.clientHeight;
-          _this3.driverRef.current.style.height = "".concat(_this3.fullPageHeight, "px");
+          _this3.viewportHeight = Math.max(document.documentElement?.clientHeight, window.innerHeight || 0);
+          _this3.fullPageHeight = _this3.fullpageRef?.current?.clientHeight;
+          console.log("I asasmade the changes. Will I see it in the browser console?");
+          _this3.driverRef?.current?.style?.height = "".concat(_this3?.fullPageHeight, "px");
           _this3.ticking = false;
         });
       }
diff --git a/node_modules/@ap.cx/react-fullpage/dist/index.js b/node_modules/@ap.cx/react-fullpage/dist/index.js
index c1b33d5..515b4c5 100644
--- a/node_modules/@ap.cx/react-fullpage/dist/index.js
+++ b/node_modules/@ap.cx/react-fullpage/dist/index.js
@@ -316,9 +316,10 @@ var Fullpage = /*#__PURE__*/function (_PureComponent) {
       if (!this.ticking) {
         window.requestAnimationFrame(function () {
           // update count
-          _this3.viewportHeight = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
-          _this3.fullPageHeight = _this3.fullpageRef.current.clientHeight;
-          _this3.driverRef.current.style.height = "".concat(_this3.fullPageHeight, "px");
+          _this3.viewportHeight = Math.max(document.documentElement?.clientHeight, window.innerHeight || 0);
+          _this3.fullPageHeight = _this3.fullpageRef?.current?.clientHeight;
+          console.log("I ssmade the changes. Will I see it in the browser console?");
+          _this3.driverRef?.current?.style?.height = "".concat(_this3?.fullPageHeight, "px");
           _this3.ticking = false;
         });
       }
